lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.html

```diff
@@ -23,12 +23,12 @@
             [field]="field"
             [displayedColumns]="displayedColumns"
             [mimeTypeIcon]="mimeTypeIcon"
-            (rowClick)="onRowClicked($event)"
-            (attachFileClick)="onAttachFileClicked($event)"
-            (downloadFile)="downloadContent($event)"
-            (uploadNewFileVersion)="onUploadNewFileVersion($event)"
-            (contentModelFileHandler)="contentModelFormFileHandler($event)"
-            (removeAttachFile)="onRemoveAttachFile($event)"
+            (rowClicked)="onRowClicked($event)"
+            (showClicked)="onViewNode($event)"
+            (downloadClicked)="onDownloadNode($event)"
+            (newVersionClicked)="onUploadNewFileVersion($event)"
+            (retrieveMetadataClicked)="onRetrieveFileMetadata($event)"
+            (removeClicked)="onRemoveFile($event)"
```

Comment:
Names like `showClicked`, `removeClicked` - does not tell me much when I see this code.

e.g. 
`showClicked` - what we are going to show? file, some more details?  I think `(attachFileClick)="..."`
looks better e.g. `(attachFileClick)=viewNode($event)`

similar the `removeClicked ` - we are going to remove property? attached file?

Reply:
I don't really get how the renaming of the events fixes the bug? seems like the refactoring that is not related to the original issue